lib/logstash/filters/pilar.rb

```diff
@@ -19,13 +19,14 @@ def register
         @gramdict = GramDict.new
         @preprocessor = Preprocessor.new(@gramdict, '<date> <time> <message>', 'message')
 
-        # populate gramdict with seed logs 
-        if @seed_logs_path 
-          ::File.open(@seed_logs_path, "r") do |seed_logs|
-            seed_logs.each_line do |seed_log|
-              # TODO: Here, we are parsing every seed log file when we don't need to, might need to separate these steps out
-              @preprocessor.process_log_event(seed_log)
-            end
+        # populate gramdict with seed logs
+        return unless @seed_logs_path
```

Comment:
Are we requiring the seed file? Should we not allow the option for no seed file (as bad as it is?)

Reply:
It is an early return, mandated by Rubocop